tests/E2E/Dockerfile

```diff
@@ -8,18 +8,20 @@ ENV PS_FOLDER_ADMIN admin-dev
 ENV PS_FOLDER_INSTALL install-dev
 ENV DB_SERVER localhost
 ENV TRAVIS_BUILD_DIR /var/www/html
+ENV DB_USER macfly
 
 # MySQL installation
 ENV DEBIAN_FRONTEND noninteractive
 RUN echo mysql-server-5.6 mysql-server/root_password password $DB_PASSWD | debconf-set-selections
 RUN echo mysql-server-5.6 mysql-server/root_password_again password $DB_PASSWD | debconf-set-selections
-RUN curl -sS https://downloads.mariadb.com/MariaDB/mariadb_repo_setup | bash -s -- --mariadb-server-version=mariadb-10.1 \
-	&& apt install -y mariadb-server
+RUN apt update \
+	&& apt install -y mysql-server
 # MySQL configuration
-RUN sed -i -e"s/^bind-address\s*=\s*127.0.0.1/bind-address = 0.0.0.0/" /etc/mysql/my.cnf
+RUN sed -i -e"s/^bind-address\s*=\s*127.0.0.1/bind-address = 0.0.0.0/" /etc/mysql/mariadb.conf.d/50-server.cnf
```

Comment:
does it have sense to rely on mariadb conf file when we now use mysql?

Reply:
MariaDb is actually installed when we ask to install the package `mysql-server`.

Reply:
oh?

Reply:
```
Step 14/29 : RUN apt update 	&& apt install -y mysql-server
 ---> Running in f0f451aef4a9

WARNING: apt does not have a stable CLI interface. Use with caution in scripts.

Get:1 http://security.debian.org/debian-security stretch/updates InRelease [94.3 kB]
Ign:2 http://cdn-fastly.deb.debian.org/debian stretch InRelease
Get:3 http://cdn-fastly.deb.debian.org/debian stretch-updates InRelease [91.0 kB]
Get:4 http://cdn-fastly.deb.debian.org/debian stretch Release [118 kB]
Get:5 http://cdn-fastly.deb.debian.org/debian stretch Release.gpg [2434 B]
Get:6 http://security.debian.org/debian-security stretch/updates/main amd64 Packages [450 kB]
Get:7 http://cdn-fastly.deb.debian.org/debian stretch-updates/main amd64 Packages [12.1 kB]
Get:8 http://cdn-fastly.deb.debian.org/debian stretch/main amd64 Packages [9530 kB]
Fetched 10.3 MB in 5s (1993 kB/s)
Reading package lists...
Building dependency tree...
Reading state information...
10 packages can be upgraded. Run 'apt list --upgradable' to see them.

WARNING: apt does not have a stable CLI interface. Use with caution in scripts.

Reading package lists...
Building dependency tree...
Reading state information...
The following additional packages will be installed:
  default-mysql-server galera-3 gawk iproute2 libatm1 libcgi-fast-perl
  libcgi-pm-perl libelf1 libencode-locale-perl libfcgi-perl
  libhtml-parser-perl libhtml-tagset-perl libhtml-template-perl
  libhttp-date-perl libhttp-message-perl libio-html-perl
  liblwp-mediatypes-perl libmnl0 libreadline7 libtimedate-perl liburi-perl
  libwrap0 libxtables12 lsof mariadb-server-10.1 mariadb-server-core-10.1
  psmisc socat tcpd
Suggested packages:
  gawk-doc iproute2-doc libdata-dump-perl libipc-sharedcache-perl libwww-perl
  mailx mariadb-test netcat-openbsd tinyca
The following NEW packages will be installed:
  default-mysql-server galera-3 gawk iproute2 libatm1 libcgi-fast-perl
  libcgi-pm-perl libelf1 libencode-locale-perl libfcgi-perl
  libhtml-parser-perl libhtml-tagset-perl libhtml-template-perl
  libhttp-date-perl libhttp-message-perl libio-html-perl
  liblwp-mediatypes-perl libmnl0 libreadline7 libtimedate-perl liburi-perl
  libwrap0 libxtables12 lsof mariadb-server-10.1 mariadb-server-core-10.1
  mysql-server psmisc socat tcpd
0 upgraded, 30 newly installed, 0 to remove and 10 not upgraded.
Need to get 14.7 MB of archives.
After this operation, 118 MB of additional disk space will be used.
Get:1 http://cdn-fastly.deb.debian.org/debian stretch/main amd64 libreadline7 amd64 7.0-3 [151 kB]
Get:2 http://cdn-fastly.deb.debian.org/debian stretch/main amd64 gawk amd64 1:4.1.4+dfsg-1 [571 kB]
Get:3 http://cdn-fastly.deb.debian.org/debian stretch/main amd64 galera-3 amd64 25.3.19-2 [955 kB]
Get:4 http://cdn-fastly.deb.debian.org/debian stretch/main amd64 libelf1 amd64 0.168-1 [157 kB]
Get:5 http://cdn-fastly.deb.debian.org/debian stretch/main amd64 libmnl0 amd64 1.0.4-2 [12.2 kB]
Get:6 http://cdn-fastly.deb.debian.org/debian stretch/main amd64 iproute2 amd64 4.9.0-1+deb9u1 [635 kB]
Get:7 http://cdn-fastly.deb.debian.org/debian stretch/main amd64 lsof amd64 4.89+dfsg-0.1 [313 kB]
Get:8 http://cdn-fastly.deb.debian.org/debian stretch/main amd64 mariadb-server-core-10.1 amd64 10.1.26-0+deb9u1 [5177 kB]
Get:9 http://cdn-fastly.deb.debian.org/debian stretch/main amd64 psmisc amd64 22.21-2.1+b2 [123 kB]
Get:10 http://cdn-fastly.deb.debian.org/debian stretch/main amd64 libwrap0 amd64 7.6.q-26 [58.2 kB]
Get:11 http://cdn-fastly.deb.debian.org/debian stretch/main amd64 socat amd64 1.7.3.1-2+deb9u1 [353 kB]
Get:12 http://cdn-fastly.deb.debian.org/debian stretch/main amd64 mariadb-server-10.1 amd64 10.1.26-0+deb9u1 [5333 kB]
Get:13 http://cdn-fastly.deb.debian.org/debian stretch/main amd64 default-mysql-server all 1.0.2 [3048 B]
Get:14 http://cdn-fastly.deb.debian.org/debian stretch/main amd64 libatm1 amd64 1:2.5.1-2 [71.0 kB]
Get:15 http://cdn-fastly.deb.debian.org/debian stretch/main amd64 libhtml-tagset-perl all 3.20-3 [12.7 kB]
Get:16 http://cdn-fastly.deb.debian.org/debian stretch/main amd64 liburi-perl all 1.71-1 [88.6 kB]
Get:17 http://cdn-fastly.deb.debian.org/debian stretch/main amd64 libhtml-parser-perl amd64 3.72-3 [104 kB]
Get:18 http://cdn-fastly.deb.debian.org/debian stretch/main amd64 libcgi-pm-perl all 4.35-1 [222 kB]
Get:19 http://cdn-fastly.deb.debian.org/debian stretch/main amd64 libfcgi-perl amd64 0.78-2 [38.2 kB]
Get:20 http://cdn-fastly.deb.debian.org/debian stretch/main amd64 libcgi-fast-perl all 1:2.12-1 [11.2 kB]
Get:21 http://cdn-fastly.deb.debian.org/debian stretch/main amd64 libencode-locale-perl all 1.05-1 [13.7 kB]
Get:22 http://cdn-fastly.deb.debian.org/debian stretch/main amd64 libhtml-template-perl all 2.95-2 [67.1 kB]
Get:23 http://cdn-fastly.deb.debian.org/debian stretch/main amd64 libtimedate-perl all 2.3000-2 [42.2 kB]
Get:24 http://cdn-fastly.deb.debian.org/debian stretch/main amd64 libhttp-date-perl all 6.02-1 [10.7 kB]
Get:25 http://cdn-fastly.deb.debian.org/debian stretch/main amd64 libio-html-perl all 1.001-1 [17.6 kB]
Get:26 http://cdn-fastly.deb.debian.org/debian stretch/main amd64 liblwp-mediatypes-perl all 6.02-1 [22.1 kB]
Get:27 http://cdn-fastly.deb.debian.org/debian stretch/main amd64 libhttp-message-perl all 6.11-1 [75.9 kB]
Get:28 http://cdn-fastly.deb.debian.org/debian stretch/main amd64 libxtables12 amd64 1.6.0+snapshot20161117-6 [75.9 kB]
Get:29 http://cdn-fastly.deb.debian.org/debian stretch/main amd64 mysql-server amd64 5.5.9999+default [1706 B]
Get:30 http://cdn-fastly.deb.debian.org/debian stretch/main amd64 tcpd amd64 7.6.q-26 [23.3 kB]
debconf: delaying package configuration, since apt-utils is not installed
Fetched 14.7 MB in 1s (7510 kB/s)
Selecting previously unselected package libreadline7:amd64.
(Reading database ... 15598 files and directories currently installed.)
Preparing to unpack .../libreadline7_7.0-3_amd64.deb ...
Unpacking libreadline7:amd64 (7.0-3) ...
Setting up libreadline7:amd64 (7.0-3) ...
Selecting previously unselected package gawk.
(Reading database ... 15611 files and directories currently installed.)
Preparing to unpack .../00-gawk_1%3a4.1.4+dfsg-1_amd64.deb ...
Unpacking gawk (1:4.1.4+dfsg-1) ...
Selecting previously unselected package galera-3.
Preparing to unpack .../01-galera-3_25.3.19-2_amd64.deb ...
Unpacking galera-3 (25.3.19-2) ...
Selecting previously unselected package libelf1:amd64.
Preparing to unpack .../02-libelf1_0.168-1_amd64.deb ...
Unpacking libelf1:amd64 (0.168-1) ...
Selecting previously unselected package libmnl0:amd64.
Preparing to unpack .../03-libmnl0_1.0.4-2_amd64.deb ...
Unpacking libmnl0:amd64 (1.0.4-2) ...
Selecting previously unselected package iproute2.
Preparing to unpack .../04-iproute2_4.9.0-1+deb9u1_amd64.deb ...
Unpacking iproute2 (4.9.0-1+deb9u1) ...
Selecting previously unselected package lsof.
Preparing to unpack .../05-lsof_4.89+dfsg-0.1_amd64.deb ...
Unpacking lsof (4.89+dfsg-0.1) ...
Selecting previously unselected package mariadb-server-core-10.1.
Preparing to unpack .../06-mariadb-server-core-10.1_10.1.26-0+deb9u1_amd64.deb ...
Unpacking mariadb-server-core-10.1 (10.1.26-0+deb9u1) ...
Selecting previously unselected package psmisc.
Preparing to unpack .../07-psmisc_22.21-2.1+b2_amd64.deb ...
Unpacking psmisc (22.21-2.1+b2) ...
Selecting previously unselected package libwrap0:amd64.
Preparing to unpack .../08-libwrap0_7.6.q-26_amd64.deb ...
Unpacking libwrap0:amd64 (7.6.q-26) ...
Selecting previously unselected package socat.
Preparing to unpack .../09-socat_1.7.3.1-2+deb9u1_amd64.deb ...
Unpacking socat (1.7.3.1-2+deb9u1) ...
Selecting previously unselected package mariadb-server-10.1.
Preparing to unpack .../10-mariadb-server-10.1_10.1.26-0+deb9u1_amd64.deb ...
Unpacking mariadb-server-10.1 (10.1.26-0+deb9u1) ...
Selecting previously unselected package default-mysql-server.
Preparing to unpack .../11-default-mysql-server_1.0.2_all.deb ...
Unpacking default-mysql-server (1.0.2) ...
Selecting previously unselected package libatm1:amd64.
Preparing to unpack .../12-libatm1_1%3a2.5.1-2_amd64.deb ...
Unpacking libatm1:amd64 (1:2.5.1-2) ...
Selecting previously unselected package libhtml-tagset-perl.
Preparing to unpack .../13-libhtml-tagset-perl_3.20-3_all.deb ...
Unpacking libhtml-tagset-perl (3.20-3) ...
Selecting previously unselected package liburi-perl.
Preparing to unpack .../14-liburi-perl_1.71-1_all.deb ...
Unpacking liburi-perl (1.71-1) ...
Selecting previously unselected package libhtml-parser-perl.
Preparing to unpack .../15-libhtml-parser-perl_3.72-3_amd64.deb ...
Unpacking libhtml-parser-perl (3.72-3) ...
Selecting previously unselected package libcgi-pm-perl.
Preparing to unpack .../16-libcgi-pm-perl_4.35-1_all.deb ...
Unpacking libcgi-pm-perl (4.35-1) ...
Selecting previously unselected package libfcgi-perl.
Preparing to unpack .../17-libfcgi-perl_0.78-2_amd64.deb ...
Unpacking libfcgi-perl (0.78-2) ...
Selecting previously unselected package libcgi-fast-perl.
Preparing to unpack .../18-libcgi-fast-perl_1%3a2.12-1_all.deb ...
Unpacking libcgi-fast-perl (1:2.12-1) ...
Selecting previously unselected package libencode-locale-perl.
Preparing to unpack .../19-libencode-locale-perl_1.05-1_all.deb ...
Unpacking libencode-locale-perl (1.05-1) ...
Selecting previously unselected package libhtml-template-perl.
Preparing to unpack .../20-libhtml-template-perl_2.95-2_all.deb ...
Unpacking libhtml-template-perl (2.95-2) ...
Selecting previously unselected package libtimedate-perl.
Preparing to unpack .../21-libtimedate-perl_2.3000-2_all.deb ...
Unpacking libtimedate-perl (2.3000-2) ...
Selecting previously unselected package libhttp-date-perl.
Preparing to unpack .../22-libhttp-date-perl_6.02-1_all.deb ...
Unpacking libhttp-date-perl (6.02-1) ...
Selecting previously unselected package libio-html-perl.
Preparing to unpack .../23-libio-html-perl_1.001-1_all.deb ...
Unpacking libio-html-perl (1.001-1) ...
Selecting previously unselected package liblwp-mediatypes-perl.
Preparing to unpack .../24-liblwp-mediatypes-perl_6.02-1_all.deb ...
Unpacking liblwp-mediatypes-perl (6.02-1) ...
Selecting previously unselected package libhttp-message-perl.
Preparing to unpack .../25-libhttp-message-perl_6.11-1_all.deb ...
Unpacking libhttp-message-perl (6.11-1) ...
Selecting previously unselected package libxtables12:amd64.
Preparing to unpack .../26-libxtables12_1.6.0+snapshot20161117-6_amd64.deb ...
Unpacking libxtables12:amd64 (1.6.0+snapshot20161117-6) ...
Selecting previously unselected package mysql-server.
Preparing to unpack .../27-mysql-server_5.5.9999+default_amd64.deb ...
Unpacking mysql-server (5.5.9999+default) ...
Selecting previously unselected package tcpd.
Preparing to unpack .../28-tcpd_7.6.q-26_amd64.deb ...
Unpacking tcpd (7.6.q-26) ...
Setting up libhtml-tagset-perl (3.20-3) ...
Setting up mariadb-server-core-10.1 (10.1.26-0+deb9u1) ...
Setting up psmisc (22.21-2.1+b2) ...
Setting up libencode-locale-perl (1.05-1) ...
Setting up libtimedate-perl (2.3000-2) ...
Setting up lsof (4.89+dfsg-0.1) ...
Setting up libelf1:amd64 (0.168-1) ...
Setting up libio-html-perl (1.001-1) ...
Setting up gawk (1:4.1.4+dfsg-1) ...
Setting up liblwp-mediatypes-perl (6.02-1) ...
Processing triggers for libc-bin (2.24-11+deb9u3) ...
Setting up libatm1:amd64 (1:2.5.1-2) ...
Setting up galera-3 (25.3.19-2) ...
Setting up libxtables12:amd64 (1.6.0+snapshot20161117-6) ...
Setting up liburi-perl (1.71-1) ...
Setting up libhtml-parser-perl (3.72-3) ...
Setting up libcgi-pm-perl (4.35-1) ...
Setting up libmnl0:amd64 (1.0.4-2) ...
Setting up libfcgi-perl (0.78-2) ...
Setting up libwrap0:amd64 (7.6.q-26) ...
Setting up libhttp-date-perl (6.02-1) ...
Setting up libhtml-template-perl (2.95-2) ...
Setting up socat (1.7.3.1-2+deb9u1) ...
Setting up iproute2 (4.9.0-1+deb9u1) ...
Setting up tcpd (7.6.q-26) ...
Setting up libcgi-fast-perl (1:2.12-1) ...
Setting up libhttp-message-perl (6.11-1) ...
Setting up mariadb-server-10.1 (10.1.26-0+deb9u1) ...
invoke-rc.d: could not determine current runlevel
invoke-rc.d: policy-rc.d denied execution of start.
Setting up default-mysql-server (1.0.2) ...
Setting up mysql-server (5.5.9999+default) ...
Processing triggers for libc-bin (2.24-11+deb9u3) ...
Removing intermediate container f0f451aef4a9
 ---> cbec69a13098
Step 15/29 : RUN sed -i -e"s/^bind-address\s*=\s*127.0.0.1/bind-address = 0.0.0.0/" /etc/mysql/mariadb.conf.d/50-server.cnf
 ---> Running in 3bbff34a0e68
Removing intermediate container 3bbff34a0e68
 ---> 97845a3243da
Step 16/29 : RUN service mysql start 	&& mysqladmin -h $DB_SERVER -P $DB_PORT -u $DB_USER -p$DB_PASSWD create $DB_NAME --force
 ---> Running in 3a1ad087e07e
Starting MariaDB database server: mysqld ..

```

Reply:
Same for Debian MariaDb is installed by default ;)